app/helpers/page_title_helper.rb

```diff
@@ -0,0 +1,9 @@
+module PageTitleHelper
+  def page_title
+    if @organisation.present?
+      @organisation.title
+    else
+      t('.title')
```

Comment:
I would not use internationalisation unless this is a requirement because would need to be consistent for every piece of text of our app. The whole process would imply more work with no clear user need.